rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -1618,7 +1618,7 @@ private byte[] generatePrivKey() {
         return privKey;
     }
 
-    private void signWithErpFederation(Federation erpFederation, List<BtcECKey> privateKeys, TransactionInput txIn, BtcTransaction tx) {
+    private void signWithErpFederation(ErpFederation erpFederation, List<BtcECKey> privateKeys, TransactionInput txIn, BtcTransaction tx) {
```

Comment:
This method is not being used. May it be removed?

Reply:
i think so... is not used in powpeg-node either @marcos-iov

Reply:
It's a privete method, so if it's not used there then it can be removed yes

Reply:
right, didnt notice its private. ill remove it, thanks guys!